core/event/event-manager.js

```diff
@@ -1642,7 +1638,7 @@ if (typeof window !== "undefined") { // client-side
                     return (this.memory[identifier] && (this.memory[identifier].size > 0));
                 },
                 storeEvent: function (mutableEvent) {
-                    var isBrowserSupportPointerEvents = this._isBrowserSupportPointerEvents,
+                    var isBrowserSupportPointerEvents = currentEnvironment.isBrowserSupportPointerEvents,
```

Comment:
This is actually a bug related to this commit:
https://github.com/montagejs/montage/commit/6004e8724594ae3c9d78ea5dc4613abb6dc92750
 
Here the context is different (pointerStorage) so the variable `isBrowserSupportPointerEvents` is always set to `undefined`.